docs/content.zh/docs/ops/state/task_failure_recovery.md

```diff
@@ -294,7 +328,22 @@ env = StreamExecutionEnvironment.get_execution_environment(config)
 
 使用群集定义的重启策略。
 这对于启用了 checkpoint 的流处理程序很有帮助。
-如果没有定义其他重启策略，默认选择固定延时重启策略。
+如果没有定义其他重启策略，默认选择指数延迟重启策略。
+
+### 默认重启策略
+
+当 Checkpoint 开启且用户没有指定重启策略时，[`指数延迟重启策略`]({{< ref "docs/ops/state/task_failure_recovery" >}}#exponential-delay-restart-strategy) 
+是当前默认的重启策略。我们强烈推荐 Flink 用户使用指数延迟重启策略并将其设置为默认重启策略，因为它相比其他重启策略可以做到: 
+作业偶尔异常时可以快速重试，作业频繁异常时可以避免外部组件发生雪崩。因为：
+
+- 所有的重启策略在重启作业时都会延迟一定的时间来避免频繁重试对外部组件的产生较大压力。
+- 除了指数延迟重启策略以外的所有重启策略延迟时间都是固定的。
+    - 如果延迟时间设置的过短，当作业短时间内频繁异常时，会频繁重启访问外部组件的主节点，可能导致外部组件发生雪崩。
+      例如：大量的 Flink 作业都在消费 Kafka，当 Kafka 集群出现故障时大量的 Flink 作业都在同一时间频繁重试，很可能导致雪崩。
+    - 如果延迟时间设置的过长，当作业偶尔失败时需要等待很久才会重试，从而导致作业可用率降低。
+- 指数延迟重启策略每次重试的延迟时间会指数递增，直到达到最大延迟时间。
```

Comment:
Maybe also mention the jitter mechanism which helps to avoid restarting jobs at the same time?